libs/openchallenges/api-description/src/paths/challenges/@{challengeId}.yaml

```diff
@@ -20,3 +20,24 @@ get:
       $ref: ../../components/responses/NotFound.yaml
     '500':
       $ref: ../../components/responses/InternalServerError.yaml
+delete:
+  tags:
+    - Challenge
+  summary: Delete a challenge
+  description: |
+    Deletes a challenge using its unique ChallengeID. This action is
+    irreversible.
+  operationId: deleteChallengeById
+  security:
+    - APIKeyAuth: []
```

Comment:
One last thing: this security schema should be defined in `challenge.openapi.yaml`.

https://learn.openapis.org/specification/security.html

```yaml
components:
  securitySchemes:
    BearerAuth:
      $ref: components/securitySchemes/ApiKeyAuth.yaml
```

Reply:
Done!